src/main/webapp/components/search-forms/container.tsx

```diff
@@ -181,6 +213,22 @@ export default () => {
     ['user', 'preferences', 'querySettings', 'template', 'id'],
     null
   )
+  const [setUserPreferences] = useSetUserPrefs()
+  const toggleDefaultForm = (id: string) => {
+    const userPrefs = {
+      querySettings: {
+        template: {
+          id: id === userDefaultForm ? null : id,
+        },
+      },
+    }
+    setUserPreferences({
+      variables: {
+        userPreferences: userPrefs,
```

Comment:
@samuelechu Should I use optimistic response here like you did: https://github.com/connexta/revelio/blob/master/src/main/webapp/components/user-settings/user-settings.js#L271

Reply:
I checked out your pr and tried switching default search forms on simulated slow 3g (took 2 seconds). The star changed index cards after the network call returned, so I'm guessing you base the default form id off the apollo cache. In that case, putting in an optimistic response makes sense, although the difference probably won't be too noticeable with faster network speeds.